modules/server/prisma.ts

```diff
@@ -1,10 +1,11 @@
 import {PrismaClient} from "@prisma/client";
 
-const {DB_1_PORT_27017_TCP_ADDR} = process.env;
+var config = require('../../config/config');
```

Comment:
IDE doesn't like var here but it looks like the rest of the code does it. Also not sure if there is a more idiomatic way of accessing global config

Reply:
Can definitely use `const` here.

I just tested using `import` instead of `require` and it does work, but it shows an IDE error about how the type of the import is unknown.

In case you were wondering, the `import` variation is `import * as config from '../../config/config';`

Let's keep the `require` syntax for importing from js files and just use `import` when importing from other ts files.